conf/cron

```diff
@@ -0,0 +1,4 @@
+# https://gitlab.com/mojo42/Jirafeau/#how-can-i-automatize-the-cleaning-of-old-expired-files
+
+12 3    * * *   __APP__  php7.0 __FINALPATH__/admin.php clean_expired
+16 3    * * *   __APP__  php7.0 __FINALPATH__/admin.php clean_async
```

Comment:
Sure it won't be some mails send to root with those lines ?

Reply:
Idk :x

Reply:
Set the cron on my own server, we'll see...

Reply:
By the way, following the bump that bring me here.
I have no files left into /home/yunohost.app/jirafeau, no mails received about it. Everything looks good.